### PR TITLE
 Add virtio-net-ccw support for s390x

### PR DESCRIPTION
**Pull request by fbq815**
virtual-network: add pci_model "virtio-net-ccw" for s390x

ID:1942866

Signed-off-by: Boqiao Fu <bfu@redhat.com>

### CODE REVIEW

**Comment by fbq815** [comment]
@PaulYuuu @ngu-niny  please help me review this patch

**Comment by fbq815** [comment]
Test result:
JOB ID     : f12b973d125aac04c3db234b809cfbe052d04fcc
JOB LOG    : /home/bfu/kar/workspace/job-results/job-2021-03-29T11.05-f12b973/job.log
 (1/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio.s390-virtio: PASS (641.15 s)
 (2/2) Host_RHEL.m9.u0.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.multi_nics_hotplug.sameid.s390-virtio: PASS (365.38 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/bfu/kar/workspace/job-results/job-2021-03-29T11.05-f12b973/results.html
JOB TIME   : 1007.91 s

**Comment by ngu-niny** [comment]
The patch depends on https://github.com/avocado-framework/avocado-vt/pull/2981

**Comment by fbq815** [comment]
@PaulYuuu since 2981 was merged, please help me review this patch

File: qemu/tests/cfg/multi_nics_hotplug.cfg [hunk 1]
```diff
@@ -22,6 +22,8 @@
                 pci_model = virtio
             nics = ""
             extra_params += "-net none"
+            s390x:
+                pci_model = virtio-net-ccw
```
Comment (PaulYuuu) [inline qemu/tests/cfg/multi_nics_hotplug.cfg:26]: As discussed at VT, ccw devices are not pci devices, so `pci_model` is not good. If you don't want to replace it in cfg and `nic_hotplug`, please add a comment or FIXME.

Comment (fbq815) [reply qemu/tests/cfg/multi_nics_hotplug.cfg:29]: add comment for it

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**